modin/logging/logger_decorator.py

```diff
@@ -105,8 +105,8 @@ def decorator(obj: Any) -> Any:
 
         assert isinstance(modin_layer, str), "modin_layer is somehow not a string!"
 
-        start_line = f"START::{modin_layer.upper()}::{name or obj.__name__}"
-        stop_line = f"STOP::{modin_layer.upper()}::{name or obj.__name__}"
+        start_line = f"STAGE: START, METHOD: {name or obj.__name__}"
```

Comment:
This is good for now but I would if we collapse START/STOP we should try to include duration.